index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Yeseva+One&display=swap"
+      rel="stylesheet"
+    />
+
+    <link rel="stylesheet" href="css/reset.css" />
+    <link rel="stylesheet" href="css/styles.css" />
+    <title>LAMBDA</title>
+  </head>
+  <body>
+    <header class="header">
+      <div class="container header-grid">
+        <dir class="logo">lambda</dir>
+
+        <nav class="nav">
+          <ul class="nav-list">
+            <li><span class="nav-item nav-item-active">home</span></li>
+            <li><span class="nav-item">about</span></li>
+            <li><span class="nav-item">ingredients</span></li>
```

Comment:
Use valid text please, no need to fix it via CSS
```suggestion
            <li><span class="nav-item nav-item-active">Home</span></li>
            <li><span class="nav-item">About</span></li>
            <li><span class="nav-item">Ingredients</span></li>
```